api/controllers/CommunityController.js

```diff
@@ -38,6 +38,21 @@ const afterCreatingMembership = (req, res, ms, community, preexisting) => {
   }))
 }
 
+const approveAJoinRequest = (req, res, joinRequest, community) => {
```

Comment:
you could also refactor this to `curry((req, res, community, joinRequest) => ...)` and then use it below with the first three args filled out, e.g. `then(approveJoinRequest(req, res, community))`